deploy/openshift/templates/multi/postgres-template.yaml

```diff
@@ -68,10 +68,10 @@ objects:
                   psql -h 127.0.0.1 -U $POSTGRESQL_USER -q -d
                   $POSTGRESQL_DATABASE -c 'SELECT 1'
             failureThreshold: 10
-            initialDelaySeconds: 30
-            periodSeconds: 5
+            initialDelaySeconds: 50
```

Comment:
I assume that these changes are wrong resolved merge conflict. If no - please describe these changes in the PR description